site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc

```diff
@@ -41,6 +42,16 @@ If you have multiple tenants listening for the same event, they will all receive
 
 include::docs/v1/tech/events-webhooks/_application-webhooks-warning.adoc[]
 
+=== Tenant Scoped Events
+
+As of version 1.37.0, all events can be tenant scoped except instance events:
```

Comment:
In the admin UI we are calling these system events. We could use the same wording here, or explain what an instance event is.